microsoft-edge/devtools-guide-chromium/javascript/consume-source-maps-from-azure.md

```diff
@@ -64,43 +104,85 @@ To generate a PAT in Azure DevOps:
+
+1. Click the **Close** (![Close settings icon](./consume-source-maps-from-azure-images/close-settings-icon.png)) button  in the upper right to close the **Settings** panel, and then click the **Reload DevTools** button.
+
 
 <!-- ====================================================================== -->
-## Step 2: Configure DevTools
+## Step 3: Retrieve original code in DevTools
 
-DevTools now needs to be configured with the personal access token (PAT) to successfully retrieve the source maps.
+After the above setup steps, when you use DevTools to work on a build of your website for which symbols have been published, you can now see your original source code, instead of the transformed code.
 
-To configure DevTools:
+*  In the **Console** tool, links from log messages to source files go to the original files, not the compiled files.
 
-1. To open DevTools, in Microsoft Edge, right-click a webpage, and then select **Inspect**.  Or, press **Ctrl+Shift+I** (Windows, Linux) or **Command+Option+I** (macOS).
+*  When stepping through code in the **Sources** tool, the original files are listed in the **Navigator** pane on the left.
 
-1. In DevTools, click **Settings** (![Settings icon](./consume-source-maps-from-azure-images/settings-gear-icon-light-theme.png)) > **Preferences**.
+*  In the **Sources** tool, the links to source files that appear in the **Call Stack** of the **Debugger** pane open the original source files.
 
-1. Make sure the **Enable JavaScript source maps** checkbox and the **Enable CSS source maps** checkbox are selected:
 
-    ![The Preferences configuration screen in DevTools settings](./consume-source-maps-from-azure-images/ado-preferences-source-maps-devtools.png)
+<!-- ====================================================================== -->
+## Connect to multiple Azure Artifacts symbol servers
+
+You can configure DevTools to connect to multiple Azure Artifacts symbol servers if you need to fetch sourcemaps from different Azure DevOps organizations for your website.
 
-1. In the sidebar of the **Settings** page, click **Symbol Server**.
+1. In DevTools, click **Settings** (![Settings icon](./consume-source-maps-from-azure-images/settings-gear-icon-light-theme.png)) > **Symbol Server**.
 
-1. In the **Azure DevOps organization** text box, enter the Azure DevOps organization where you created the PAT.
+1. Click **Add Symbol Server connection** to create an additional connection.
 
-1. In the **Azure DevOps personal access token** text box, paste your personal access token (PAT).
+1. Refer to [Connect to the symbol server with Azure Active Directory](#connect-to-the-symbol-server-with-azure-active-directory) or [Connect to the symbol server with a Personal Access Token](#connect-to-the-symbol-server-with-a-personal-access-token) to configure the new connection.
```

Comment:
```suggestion
1. Configure the new connection.  See [Connect to the symbol server with Azure Active Directory](#connect-to-the-symbol-server-with-azure-active-directory) or [Connect to the symbol server with a Personal Access Token](#connect-to-the-symbol-server-with-a-personal-access-token).
```
Antipattern:
Do X, Y, and Z, to accomplish A.
Pattern (applies in some cases):
To accomplish A, do X, Y, and Z.